### PR TITLE
Fixing the link for kfdef

### PR DESCRIPTION
**Pull request by nakfour**
Fixing the broken link to the Openshift kfdef

### CODE REVIEW

File: content/en/docs/distributions/openshift/install-kubeflow.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Minimal:
 Use the following steps to install Kubeflow 1.0 on OpenShift 4.x.
 
 1. Download the example "kfdef" for Kubeflow 1.3 on Openshift from [kubeflow/manifests/distributions/kfdef]
-(https://raw.githubusercontent.com/kubeflow/manifests/master/distributions/kfdef/kfctl_openshift_v1.3.0.yaml).
+(https://github.com/opendatahub-io/manifests/blob/v1.3-branch/distributions/kfdef/kfctl_openshift_v1.3.0.yaml).
```
Comment (yhwang) [inline content/en/docs/distributions/openshift/install-kubeflow.md:43]: should it be: https://raw.githubusercontent.com/opendatahub-io/manifests/v1.3-branch/distributions/kfdef/kfctl_openshift_v1.3.0.yaml ?

**Review by Arhell — COMMENTED** [review]
please resolve conflicts

File: content/en/docs/distributions/openshift/install-kubeflow.md (hunk 1 above)
Comment (nakfour) [reply content/en/docs/distributions/openshift/install-kubeflow.md]: thanks @yhwang changed it to raw.

File: content/en/docs/distributions/openshift/install-kubeflow.md [hunk 2]
```diff
@@ -39,7 +39,10 @@ Minimal:
 
 Use the following steps to install Kubeflow 1.0 on OpenShift 4.x.
```
Comment (yhwang) [inline content/en/docs/distributions/openshift/install-kubeflow.md:40]: not related to your change, but notice it says Kubeflow `1.0` here.

Comment (nakfour) [reply content/en/docs/distributions/openshift/install-kubeflow.md]: yes fixed it

**Review by yhwang — APPROVED** [review]

**Pull request merged**